frontend/src/shared/components/tasque-task-editing/task-editing.component.ts

```diff
@@ -178,17 +179,20 @@ export class TaskEditingComponent extends BaseComponent implements OnInit {
     {
       id: 1,
       name: 'User1Name',
-      email: 'email'
+      email: 'email',
+      organizationRoles: [ { organizationId: 1, userId: 2, role: UserRole.organizationMember }, { organizationId: 2, userId: 2, role: UserRole.organizationMember } ]
     },
     {
       id: 2,
       name: 'User2Name',
-      email: 'email'
+      email: 'email',
+      organizationRoles: [ { organizationId: 1, userId: 2, role: UserRole.organizationMember }, { organizationId: 2, userId: 2, role: UserRole.organizationMember } ]
     },
     {
       id: 3,
       name: 'User3Name',
-      email: 'email'
+      email: 'email',
+      organizationRoles: [ { organizationId: 1, userId: 2, role: UserRole.organizationMember }, { organizationId: 2, userId: 2, role: UserRole.organizationMember } ]
     }
```

Comment:
Do we still need this?

Reply:
> Do we still need this?

I dont know, need to ask someone who is doing this functionality. If this dev has finished work on it I might change it's logic according to new features, but I think it's better to do in another branch. Otherwise, if he's still working on it I can ask him to remove this values

Reply:
I have already removed it in my branch, so you can remove it here too

Reply:
> I have already removed it in my branch, so you can remove it here too

👍